### PR TITLE
fix: make all margins consistent across pages

### PR DESCRIPTION
**Pull request by carissa-tang**
## Implementation description. How did you make this change?
* Fixed margins to be consistent throughout pages
* Header should be inline for mobile view (`Samsung Galaxy S8+ [360x740]`)

## Steps to test
1. Check margins / page layout on Donor View
2. Check on Admin View

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages follow conventional commits and are descriptive. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s) 
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
- [ ] The appropriate tests if necessary have been written

### CODE REVIEW

File: frontend/src/components/common/Footer.tsx [hunk 1]
```diff
@@ -7,12 +7,11 @@ const Footer = (): JSX.Element => (
   <>
     <Divider pt={{ base: "79px", md: "117px" }} />
     <Container
-      maxWidth={{ base: "default", md: "70%", lg: "80%" }}
+      maxWidth={{ lg: "80%" }}
```
Comment (hanlinc27) [inline frontend/src/components/common/Footer.tsx:10]: the footer seems to still be misaligned for me: 
my guess is probably to change the `justifyContent="space-evenly"` to `space-between` and then adjust the width back? 
image

**Review by hanlinc27 — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/Footer.tsx (hunk 1 above)
Comment (hanlinc27) [reply frontend/src/components/common/Footer.tsx:10]: or instead of adjusting width back, pass it the same variant as the header?

Comment (carissa-tang) [reply frontend/src/components/common/Footer.tsx]: made a fix! could you double check to see if it's what you want?

**Review by hanlinc27 — APPROVED** [review]
giphy (4)

File: frontend/src/components/common/Footer.tsx (hunk 1 above)
Comment (hanlinc27) [reply frontend/src/components/common/Footer.tsx]: looks great! 😄

**Pull request merged**